Resources/Prototypes/_Impstation/Entities/Mobs/Species/apid.yml

```diff
@@ -0,0 +1,265 @@
+- type: entity
+  save: false
+  name: Urist McBee
+  parent: BaseMobSpeciesOrganic
+  id: BaseMobApid
+  abstract: true
+  components:
+  - type: Absorbable
```

Comment:
could you add `reducedBiomass: true` to this, similar to goblins?

Reply:
or do this in a tiny little pr in the future because kip is very upset that bees arent getting caught in this publish